### DISCUSSION TITLE
همکاری

### DISCUSSION THREAD

**Discussion by peter19822003** [GFW-knocker/Segaro_Dream#6] [General]
با نام خدای رنگین کمان
بسیار ایده خوب و عالی هست و من بو نوبه خود حمایت و همکاری خود را اعلام میکنم
بسیار خوشحال میشوم بتونم به شما در این پروژه عالی کمک کنم

**Comment by GFW-knocker** [comment]
بسیار عالی. ممنون.

**Reply by mahxray** [reply]
سلام سگارو جان
فایل کتابخونه libv2ray.aar رو میتونی در اختیارمون بزاری ؟؟

**Comment by GFW-knocker** [comment]
> سلام سگارو جان فایل کتابخونه libv2ray.aar رو میتونی در اختیارمون بزاری ؟؟

اخرین نسخه فایل arr اینجاست
https://github.com/yuhan6665/v2rayNG/releases/download/aar-test/libv2ray.aar
https://github.com/yuhan6665/v2rayNG/releases/tag/aar-test

دوست عزیزمون سگارو ایشون هست و بنده از مریدانشونم فقط
https://github.com/iSegaro
https://twitter.com/isegaro

**Reply by mahxray** [reply]
ممنونم
